### PR TITLE
KYODO Impression revenue model

### PR DESCRIPTION
**Pull request by TurboUrbo**
Paragraph is added
@igorline please tell what to add  or remove

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,25 @@ HARRY HALPIN Research Scientist at Massachusetts Institute of Technology [↩](#
...
+> Now that the commodity value of human labor is becoming increasingly tangential in an even more automated world, new ways of defining human worth and the social relationship will need to be explored
+>
+> -- *Jeremy Rifkin* "End of work"
+
+"Pay per hour" or monthly salary better fits for robots that produce countable units with a stable quality. 
+How the enthusiasm or dedication could be evaluated?
+How the progressiveness or intuition could be estimated?
+How the single time but relevant contribution could be rewarded?
+
+KYODO suggests Impression revenue model. The contribution is evaluated by members of the community in the momentum. The Nash Equilibrium is achieved through the Token economy, where the token value is based on the activity of the members. Means that every member of a community is interested to evaluate other contributors fair and motivate them to further actions. Also, the group members know the best if someone did a good job.
```
Comment (igorline) [inline README.md]: Maybe token value is based on the effectiveness of the activity of the members?
Actually as it is asset backed the price is based on the cap of the fund, but holders/members interests are aligned to increase the token price by increasing its cap

File: README.md [hunk 2]
```diff
@@ -60,6 +60,25 @@ HARRY HALPIN Research Scientist at Massachusetts Institute of Technology [↩](#
...
+
+"Pay per hour" or monthly salary better fits for robots that produce countable units with a stable quality. 
+How the enthusiasm or dedication could be evaluated?
+How the progressiveness or intuition could be estimated?
+How the single time but relevant contribution could be rewarded?
+
+KYODO suggests Impression revenue model. The contribution is evaluated by members of the community in the momentum. The Nash Equilibrium is achieved through the Token economy, where the token value is based on the activity of the members. Means that every member of a community is interested to evaluate other contributors fair and motivate them to further actions. Also, the group members know the best if someone did a good job. 
+More about DF token model read in Token economy paragraph. (link)
+
+What is even cooler is that Impression revenue model is instant tipping that is adapted for the working platform that the community is using (Riot, for now, GitHub and Jitsi in progress).
```
Comment (igorline) [inline README.md]: > What is even cooler...

This format fits better article rather than paper

File: README.md [hunk 3]
```diff
@@ -73,11 +73,11 @@ How the single time but relevant contribution could be rewarded?
...
+What is even cooler is that Impression revenue model is instant tipping that is adapted for the working platform that the community is using (Riot, for now, Github and Jitsi in progress).
 How the instant tipping works:
-- A member posts a link to a Riot that some contribution was done (twitter post or GitHub code)
-- Other members check the contribution and call the bot, asking to send the tip to the contributor. (example)
-- The earned tips are collected automatically within a period. The earned tips are converted to DF token at the end of a period and transferred to the member private address. 
+- A member shares to Riot details on the contribution he did explaining its importance for the community (Twitter post or Github code)
+- Other members check the contribution and using connected bot send tips to the contributor.
+- The earned tips are collected automatically within a period. The earned tips are converted to DF token at the end of a period and transferred to the member ethereum address.
```
Comment (igorline) [inline README.md]: @TurboUrbo fixed this part if you don't mind

**Comment by igorline** [comment]
@TurboUrbo, I've added comments

File: README.md (hunk 1 above)
Comment (TurboUrbo) [reply README.md]: agree

**Comment by igorline** [comment]
@TurboUrbo, I've merged master branch with inflation part by @pasha0x.
Please, review

**Review by igorline — CHANGES_REQUESTED** [review]
@TurboUrbo review latest version

**Review by igorline — APPROVED** [review]

**Pull request merged**